index.js

```diff
@@ -3,13 +3,18 @@
 'use strict';
 
 require('shelljs/global');
-var moment = require('moment');
-var _ = require('lodash');
-var app = require('./package');
+
+const os = require('os'); // access the operating system type
```

Comment:
All good! Can we remove the comments for the modules please?

Reply:
Committed.